### DISCUSSION TITLE
Update `buffer` dependency to v6

### DISCUSSION THREAD

**Discussion by feross** [vercel/next.js#32088] [Ideas]
**Describe the feature you'd like to request**

I'm the maintainer of [`buffer`](https://github.com/feross/buffer). The version used by `next` is v5 which contains bugs and performance issues which are resolved by `buffer` v6.

**Describe the solution you'd like**

Can you consider updating the dependency to the latest version so that users can get the latest fixes? Thanks!

**Describe alternatives you've considered**

I assume there's some way to override the webpack config to force a newer version of `buffer` to be used, but I haven't figured out how to do that yet.

**Comment by ludofischer** [comment]
Seems hard to update for now, since Next.js [supports IE 11](https://nextjs.org/docs/basic-features/supported-browsers-features) and buffer 6 [dropped IE 11 support ](https://github.com/feross/buffer/commit/840160f352ff0d8e88d4ac308a1640fd278f50fc).
